RetailCoder.VBE/UI/CodeExplorer/Commands/ImportCommand.cs

```diff
@@ -67,6 +69,11 @@ protected override void OnExecute(object parameter)
 
         private IVBProject GetNodeProject(CodeExplorerItemViewModel parameter)
         {
+            if (parameter == null)
```

Comment:
parameter is a kind of bad name ... also why is `null` somewhat expected now?

Reply:
The import command was designed to have an item selected in the TreeView when activated.  It makes sense if there is more than one project; that way the user is choosing which project to import a module into.  However, it did not handle the use case where a node was not selected.  Handling null fixes that.

Reply:
`parameter` is still a terrible name 😉 

We're looking at `IVBProject GetNodeProject`, which implies we're getting a *project* out of a *node*... how about this?

```csharp
private IVBProject GetNodeProject(CodeExplorerItemViewModel node)
```

Reply:
I agree that parameter is a bad name.  But if I change it, I might as well change it in every other method in the class. 😑